This patch adds Solaris as a supported OS.  This patch has been
submitted upstream but will not be accepted until Solaris 12 is
released publicly AND a Puppetlabs customer requests its support for
Puppet Enterprise.

puppetlabs/apt dependency removal is not appropriate for upstream. It removes
a warning from puppet module list.
diff --git a/README.md b/README.md
index 358f953..e143553 100644
--- a/README.md
+++ b/README.md
@@ -1,3 +1,5 @@
+NOTE: Oracle has modified the originally distributed contents of metadata.json.
+
 #rabbitmq

 ####Table of Contents
diff --git a/metadata.json b/metadata.json
index bdc249e..aaf4e44 100644
--- a/metadata.json
+++ b/metadata.json
@@ -35,6 +35,14 @@
         "12.04",
         "14.04"
       ]
+    },
+    {
+      "operatingsystem": "Solaris",
+      "operatingsystemrelease": [
+        "11.3",
+        "11.4",
+        "12.0"
+      ]
     }
   ],
   "requirements": [
@@ -49,7 +57,6 @@
   ],
   "dependencies": [
     {"name":"puppetlabs/stdlib","version_requirement":">=3.0.0 <5.0.0"},
-    {"name":"puppetlabs/apt","version_requirement":">=1.8.0 <3.0.0"},
     {"name":"nanliu/staging","version_requirement":">=0.3.1 <2.0.0"}
   ]
 }
